payu_mobile_payments/payu_mobile_payments_platform_interface/lib/models/android/request/payment_method_tokenization_specification_parameters.dart

```diff
@@ -1,17 +1,23 @@
 class PaymentMethodTokenizationSpecificationParameters {
-  final String gateway;
-  final String gatewayMerchantId;
+  final String? gateway;
```

Comment:
czemu to nie jest typu String tylko String? wydaje mi się że to są krytyczne dane do poprawnego działania